src/flows/templates/types/demo-data

```diff
@@ -0,0 +1,199 @@
+{
+  "name": "Demo Data Notebook",
```

Comment:
We should change this name to be more descriptive of what type of demo purpose this notebook serves.